File: src/main/java/catwalks/part/IDecoratable.java
package catwalks.part;

import catwalks.block.EnumDecoration;

/**
 * Created by TheCodeWarrior
 */
public interface IDecoratable {
	boolean addDecoration(EnumDecoration decor);
	boolean removeDecoration(EnumDecoration decor);
	boolean hasDecoration(EnumDecoration decor);
}

File: src/main/java/catwalks/part/IDecoratable.kt
package catwalks.part

import catwalks.EnumDecoration

/**
 * Created by TheCodeWarrior
 */
interface IDecoratable {
    fun addDecoration(decor: EnumDecoration): Boolean
    fun removeDecoration(decor: EnumDecoration): Boolean
    fun hasDecoration(decor: EnumDecoration): Boolean
}
